CHANGES.rst

```diff
@@ -4,6 +4,8 @@ Release 0.3
 Major changes
 -------------
 
+* New feature: joining tables using :func:`fuzzy_join` by specifying the two columns on which
+    the join will be perfomed.
```

Comment:
This description sounds like that of a standard join. We should mention that the join is done by approximate key matching.

Reply:
I hope it's clearer now.